### PR TITLE
Bryce/update aws tile reference

### PR DESCRIPTION
**Pull request by buraizu**
### What does this PR do?
- Update references to the AWS integration tile
- Replace outdated screenshots
- Minor copy edits

### Motivation
DOCS-3898

### Previews

https://docs-staging.datadoghq.com/bryce/update-aws-tile-reference/account_management/billing/aws/#aws-resource-exclusion

https://docs-staging.datadoghq.com/bryce/update-aws-tile-reference/integrations/guide/aws-integration-and-cloudwatch-faq/

https://docs-staging.datadoghq.com/bryce/update-aws-tile-reference/integrations/guide/error-datadog-not-authorized-sts-assume-role/

https://docs-staging.datadoghq.com/bryce/update-aws-tile-reference/integrations/guide/monitor-your-aws-billing-details/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by kayayarai — APPROVED** [review]
You might want to consider not deleting the old images, as the translated pages will still try to use them until the translation pipeline catches up. We generally do a scan of unused images every few months to delete them after they aren't used anymore.

File: content/en/account_management/billing/aws.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ To filter out other resources use the [API][8].
 
 ## AWS resource exclusion
 
-Use the [Datadog-AWS integration tile][3] to control your metric collection. Go to the **Configuration** tab and select an account or add a new one. Each account is controlled under **Optionally limit resource collection**. Limit metrics by [host tag][4], lambda tag, or per namespace:
+Use the [Datadog-AWS integration page][3] to control your metric collection. Select the AWS account and click on the **Metric Collection** tab. Under **Limit Metric Collection to Specific Resources** you can exclude metrics by [host tag][4], Lambda tag, or namespace:
```
Comment (warrierr) [inline content/en/account_management/billing/aws.md:22]:
```suggestion
You can limit the AWS metrics collected for some services to specific resources. On the [Datadog-AWS integration page][3], select the AWS account and click on the **Metric Collection** tab. Under **Limit Metric Collection to Specific Resources** you can then exclude metrics for one or more of EC2, Lambda, ELB, Application ELB, Network ELB, RDS, SQS, and CloudWatch custom metrics.
```

File: content/en/account_management/billing/aws.md [hunk 2]
```diff
@@ -19,15 +19,15 @@ To filter out other resources use the [API][8].
...
 
-{{< img src="account_management/billing/aws02.png" alt="AWS" >}}
+{{< img src="account_management/billing/aws-resource-exclusion.png" alt="The metric collection tab of an AWS account within the Datadog AWS integration page showing the option to limit metric collection to specific resources with a dropdown menu to select AWS service and a field to add tags in key:value format" >}}
 
 **Note**: Datadog does not charge for ELB metrics, as they can’t be filtered out.
```
Comment (warrierr) [inline content/en/account_management/billing/aws.md:26]:
```suggestion
**Note**: Only EC2 (hosts), Lambda (active functions), and CloudWatch Custom Metrics (custom metrics) are billable by Datadog. Metrics integrated for the other services you can filter do not incur Datadog charges.
```

File: content/en/account_management/billing/aws.md [hunk 3]
```diff
@@ -19,15 +19,15 @@ To filter out other resources use the [API][8].
...
 
 **Note**: Host resource exclusion settings apply to both EC2 and its attached EBS volumes.
```
Comment (warrierr) [inline content/en/account_management/billing/aws.md:28]:
```suggestion
**Note**: EC2 metrics resource exclusion settings apply to both EC2 and its attached EBS volumes. 
```

**Review by warrierr — CHANGES_REQUESTED** [review]
Just these few adjustments

**Review by warrierr — APPROVED** [review]

**Pull request merged**